.github/workflows/perf-compare-ci.yml

```diff
@@ -0,0 +1,215 @@
+name: Binding Perf Tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      pull_request:
+        description: The pull request number to run the perf tests. The workflow compares the specified pull request with the trunk.
+        required: true
+
+jobs:
+    jikesrvm-perf-compare:
+        runs-on: [self-hosted, Linux, freq-scaling-off]
+        steps:
+          - run: echo "PERF_PR=${{ github.event.inputs.pull_request }}" >> $GITHUB_ENV
+          - name: Check Revisions
+            uses: qinsoon/comment-env-vars@1.1.0
+            with:
+              token: ${{ secrets.GITHUB_TOKEN }}
+              pull_request: ${{ env.PERF_PR }}
+              default_env: 'JIKESRVM_BINDING_TRUNK_REF=master,MMTK_CORE_TRUNK_REF=master,JIKESRVM_BINDING_BRANCH_REF=master,MMTK_CORE_BRANCH_REF=${{ github.event.pull_request.head.sha }}'
+          # Trunk
+          # - binding
+          - name: Checkout JikesRVM Binding Trunk
+            uses: actions/checkout@v2
+            with:
+              repository: mmtk/mmtk-jikesrvm
+              token: ${{ secrets.GITHUB_TOKEN }}
+              path: mmtk-jikesrvm-trunk
+              submodules: true
+              ref: ${{ env.JIKESRVM_BINDING_TRUNK_REF }}
+          # - core
+          - name: Checkout MMTk Core
+            uses: actions/checkout@v2
+            with:
+              ref: ${{ env.MMTK_CORE_TRUNK_REF }}
+              path: mmtk-core-trunk
+          # Branch
+          # - binding
+          - name: Checkout JikesRVM Binding Branch
+            uses: actions/checkout@v2
+            with:
+              repository: mmtk/mmtk-jikesrvm
+              token: ${{ secrets.GITHUB_TOKEN }}
+              path: mmtk-jikesrvm-branch
+              submodules: true
+              ref: ${{ env.JIKESRVM_BINDING_BRANCH_REF }}
+          # - core
+          - name: Checkout MMTk Core
+            uses: actions/checkout@v2
+            with:
+              ref: ${{ env.MMTK_CORE_BRANCH_REF }}
+              path: mmtk-core-branch
+          # Checkout perf-kit
+          - name: Checkout Perf Kit
+            uses: actions/checkout@v2
+            with:
+              repository: mmtk/ci-perf-kit
+              token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Aren't we supposed to use `CI_ACCESS_TOKEN` to clone private submodules?

Reply:
Yes. The version of `ci-perf-kit` in this PR does not use private submodules, so it's fine. I can change it here though.

Reply:
Ah, right. That can be fixed in #444. No problem. Just leave it as is.